pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -711,12 +734,16 @@ func getAttribute(key, val string) (*eosclient.Attribute, error) {
 }
 
 // GetFileInfoByPath returns the FilInfo at the given path.
-func (c *Client) GetFileInfoByPath(ctx context.Context, auth eosclient.Authorization, path string) (*eosclient.FileInfo, error) {
+func (c *Client) GetFileInfoByPath(ctx context.Context, userAuth eosclient.Authorization, path string) (*eosclient.FileInfo, error) {
 	log := appctx.GetLogger(ctx)
-	log.Debug().Str("func", "GetFileInfoByPath").Str("uid,gid", auth.Role.UID+","+auth.Role.GID).Str("path", path).Msg("entering")
+	log.Debug().Str("func", "GetFileInfoByPath").Str("uid,gid", userAuth.Role.UID+","+userAuth.Role.GID).Str("path", path).Msg("entering")
```

Comment:
I will remove it from the log line because indeed it's useless here. But it's part of the signature because this method is defined in the eos client interface, and it's used in the binary client so we can't drop it

Reply:
We could also look at the eosbinary code, and eventually change the signature - that's not forbidden!

Reply:
Following @diocas 's other comment, I changed it to now do this as the user if he has a uid set, and otherwise as `daemon`